lib/plugins/aws/package/compile/events/apiGateway/lib/stage.js

```diff
@@ -98,8 +97,7 @@ module.exports = {
       // delete cfTemplate.Resources[deploymentId].Properties.StageName;
       // --------------------------------------------------------------------------------
     }
-
-    return BbPromise.resolve();
+    return undefined;
```

Comment:
Is it needed to return explicit `undefined` here? If that's the case, I think it's better to leave `BbPromise.resolve()`, because returning `undefined` feels unnatural

Reply:
Yeah I agree, are you okay with just 
```diff
- return undefined;
+ return;
```

Reason being, when this is converted to async/await, line 90 will probably do something like the following:
```diff
- return ensureApiGatewayCloudWatchRole(this.provider);
+ await ensureApiGatewayCloudWatchRole(this.provider);
```
Or something like that anyway. And then that way you have your `void` function ready to go. What do you think?

Reply:
Only caveat there is that that will violate the `consistent-return` lint rule.

Reply:
To keep the refactoring minimal, I think returning `Promise.resolve` here will be okay, and during further refactoring we can potentially clean that up and introduce more changes. :+1: